### PR TITLE
Update README.md

### LINKED ISSUE #26: “rustc” cannot be opened because the developer cannot be verified.
Screenshot 2024-04-30 at 10 39 09 PM

I guess some signing will be needed for prebuild version?

### LINKED ISSUE #27: build instruction unclear
```
$ ./build.sh --DCMAKE_BUILD_TYPE=Release
config.sh: line 14: cd: rust: No such file or directory
```

cloned rust repo and `cp config.example.toml config.toml ` and getting 

```
$ ./build.sh --DCMAKE_BUILD_TYPE=Release
Building bootstrap
warning: `/Users/xiliangchen/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/Users/xiliangchen/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
    Finished `dev` profile [unoptimized] target(s) in 0.13s
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 123711` at the top of `config.toml`
thread 'main' panicked at src/core/sanity.rs:203:13:
No such target exists in the target list,
                specify a correct location of the JSON specification file for custom targets!
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Build completed unsuccessfully in 0:00:00
```

### PR DESCRIPTION
**Pull request by athei**
Fixes #26
Fixes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,39 @@
 
 This repository compile and publish as release final binaries of the Rust toolchain targetting RISC-V RV32E.
 
-## Download 
+## Compiling
+
+```sh
+./clone.sh
+./build.sh
```
Comment (xlc) [inline README.md:10]: do we not need patch before build?

File: README.md [hunk 2]
```diff
@@ -2,19 +2,39 @@
...
+./package.sh
+```
+
+Executing those steps will create a `rust-rve-nightly-$rust_commit_date-$TOOLCHAIN_HOST_TRIPLET.xz` in your working directory.
```
Comment (xlc) [inline README.md]: should also describe how do we link this with rustup

Comment (athei) [reply README.md]: Its covered under `#Installation` but I added more explanation now.

File: README.md (hunk 1 above)
Comment (athei) [reply README.md:10]: Yes. Good catch.

**Pull request merged**